package.json

```diff
@@ -5,7 +5,10 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint",
+    "lint:check": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "lint-changes:check": "eslint -c .eslintrc_strict.js $(git diff --name-only --diff-filter=ACMRTUXB origin/dev | grep  -E \"(.js$|.vue$)\")",
+    "lint-changes:fix": "eslint -c .eslintrc_strict.js $(git diff --name-only --diff-filter=ACMRTUXB origin/dev | grep  -E \"(.js$|.vue$)\") --fix",
```

Comment:
These scripts are used for locally linting only the files that differ from origin/dev. A nice explanation for the bash can be found here: https://gist.github.com/seeliang/0f0de424d1cdc4541c338f4ee93b7e6a

I'm not perfectly happy that this isn't exactly the same as the process that runs in github actions. The latter uses a pre-existing action from the marketplace because I couldn't get this script to work with the hardcoded comparison branch (maybe it's possible, I didn't spend much time on it). So this won't work as a way of locally checking the strict linter on changes with a branch that's not dev. 

Worst case if it fails, though, is you find some linting issues when the github action runs and then you can fix and resubmit. Or manually run the strict linter on your changes. I think it's an ok compromise for now, and we can automate it later if it ends up causing pain.